### ISSUE TITLE
Can i help you 

### ISSUE THREAD

**Issue by ghost** [MineCode-Devs/DevMine#6]
@remotevase  @AmiroEmad

**Comment by MiroTheSuperDev** [comment]
Sure

But please tell what are you willing to help in

**Comment by remotevase** [comment]
of course! of you can fix the blank screen problem, them that would be amazing! ooh, and if you do, please upload it as a zip via pull requests. thanks! if you can fix this, then you proved yourself worthy of being a collaborator.
